app/Http/Controllers/ItemsController.php

```diff
@@ -70,8 +70,18 @@ public function unfaveItem(Request $request) {
     }
 
 
-    public function showUserHome() {
-        return view('welcome');
+    public function showUserHome(Request $request) {
+        $categoryCount = DB::table('items')
+          ->join('categories', 'items.category_id', '=', 'categories.id')
```

Comment:
I'm unfamiliar with Eloquent (and laravel as a whole, frankly), so I was just aiming for a working implementation of the basic concept. Definitely would benefit from some refinement. :)